### DISCUSSION TITLE
REQ: Make /teamlock affect 1on1 (regarding skins)

### DISCUSSION THREAD

**Discussion by suddendeathTV** [ezQuake/ezquake-source#881] [Ideas]
Hey,

in TDM, the command `teamlock 1` allows you to use `teamskin` and `enemyskin` to define skins of two separate teams as a third party (observer). This is especially useful for streaming where you may want one team to always remain one skincolor and the other team to remain the other teamcolor when either floating around the map in specmode or when hopping between players' perspectives.

There is currently no similar function for 1on1/duel, except using `cl_name_as_skin`, but in a tournament of 100+ players that solution requires hundreds of individual skins that completely break if someone changes their nickname.

Request: make `teamlock` affect 1on1/duel in the same way it affects TDM. Technically this might be more difficult than it sounds depending on how the server treats the game modes (I have no idea), but I'm sure someone is smart enough for it...

Hope someone could grab this request!

**Comment by VladGldyrojer** [comment]
Ah, I get what you’re saying. That sounds like a pretty big hassle, especially for tournaments with a lot of players. The cl_name_as_skin workaround definitely doesn’t scale well when you’ve got 100+ players, and with nicknames Ключ вкладка info https://casinonic.casinologin.mobi/ changing, it could cause a lot of headaches for streamers and observers trying to keep track of everyone.

**Comment by JesseReed123** [comment]
Je comprends votre demande. Actuellement, teamlock 1 fonctionne bien en TDM pour forcer les skins des équipes, ce qui est pratique pour le streaming et garder une distinction claire entre les équipes. Pour les matchs 1v1 ou duel, il n’existe pas de fonctionnalité équivalente, et cl_name_as_skin est très difficile à gérer pour un tournoi de 100+ joueurs. Étendre teamlock aux [iledecasino ](https://ile-de-casino.pro/)duels permettrait de standardiser les skins et simplifier le suivi visuel pour les spectateurs, mais cela pourrait demander des ajustements techniques côté serveur. C’est une suggestion pertinente pour améliorer l’expérience compétitive et le streaming._
